### ISSUE TITLE
Thêm 1 vài tính năng

### ISSUE THREAD

**Issue by thanhps42** [ProxymanApp/Proxyman#504]
Mình đã nhìu năm xài Fiddler và Charles (charles xài rất chán nên mình chỉ mua lic nhưng hiếm khi xài nó).

Sau vài năm xài Fiddler mình thấy Fiddler có những cái hay như sau:
- Tính năng Mark có hỗ trợ phím tắt (mình có ghi chi tiết bên dưới).
- Phím tắt để on/off Capture.
- Có 1 cái mình thấy khá bất tiện khi xài Repeat là mình phải on capture lên (lúc này nhìu app nó gửi request nên cái repeat của mình bị lạc trôi mất), nên mình hy vọng là khi Repeat thì dù đã off capture mình vẫn bắt được nó; vì khi repeat là mình đã tập trung vào 1 request duy nhất chứ ko muốn bị phân tâm bởi những request khác.
- Mỗi response content-type sẽ có 1 màu cho dễ phân biệt.

Về chức năng Mark:
Mình thấy chức năng tô màu 1 dòng request của Fiddler rất hay, hy vọng bạn có thể thêm vào được. Vì pinned mình xài thực sự cảm thấy không tiện bằng tô màu như vầy, tô màu thì dạng như đánh dấu 1 cái gì đó tạm lại khác với pinned là lưu giữ mãi. Với hy vọng bạn có thể thêm phím tắt để thao tác cho nhanh.

![](https://i.postimg.cc/056rVzb7/screenshot1.png)

P/s: Cám ơn bạn rất nhiều vể ProxyMan, suốt mấy năm qua mình xài mac nhưng rất ức chế với Charles, toàn phải lên máy windows để xài Fiddler. Nay có proxyman thì ko cần máy Windows nữa rồi :D

**Comment by NghiaTranUIT** [comment]
Hey, cảm ơn bạn đã report mấy features từ Fiddler.
- Mark: Cái này rất hữu dụng 
- Đúng là mình thiếu hotkey để ON/OFF Recording. Hiện tại chỉ có ON/OFF Proxy (CMD+SHIFT+OPTION+O)
- Bạn có thể thử Repeat bằng CMD+Enter mà ko cần phải "on Capture". Ngoài ra, để mình tìm thêm 1 option vào phần filter, để filter những request đã được Edit (Repeat, Map Local, Remote,...)
- Về "Màu cho từng loại Content-Type", mình sẽ cân nhắc, vì ko muốn app sẽ bị loạn màu, nhất là trong Dark-theme 😅

**Comment by thanhps42** [comment]
Màu content-type mặc định cho off cũng dc bạn, mà có option kích hoạt trong Preferences thì ngon :3

**Comment by NghiaTranUIT** [comment]
Good point 👍  or it could be a plugin that I'm working on. It means it's flexible to customize as your personal taste

**Comment by NghiaTranUIT** [comment]
Hey @thanhps42 and @suho,

Tin vui là mình đã code feature này 😄 

--------------------------------

All done with BETA: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_2.11.1_Highlight_Color_Add_Comment.dmg

#### Shortcuts List
+ Highlight Color with CMD+Number
+ Strikethrough by CMD+/
+ Reset by CMD+0

![Screen_Shot_2020-11-08_at_21_44_32](https://user-images.githubusercontent.com/5878421/98549359-410dde00-22cd-11eb-89fc-d78d737e44ff.png)

<img width="1038" alt="Screen Shot 2020-11-09 at 20 50 06" src="https://user-images.githubusercontent.com/5878421/98549320-33f0ef00-22cd-11eb-8842-649db7f40036.png">

**Comment by NghiaTranUIT** [comment]
Sorry vì mình tập trung vào v2 Proxyman (Code lại cái Core với Swift NIO, với thêm 1 vài feature quan trọng như Scripting, và [Atlantis](https://github.com/ProxymanApp/atlantis), ....)

Mình sẽ release sớm vào tuần này 🙌
